docs/docs/gatsby-cli.md

```diff
@@ -57,6 +57,23 @@ Once you've installed a Gatsby site, go to the root directory of your project an
 Follow the [Local HTTPS guide](/docs/local-https/)
 to find out how you can set up an HTTPS development server using Gatsby.
 
+#### Preview changes on other devices
+
+You can use the Gatsby develop command with the host option to access your dev environment on other devices on the same network, run:
+
+```shell
+gatsby develop -H 0.0.0.0
+```
+
+Then the terminal will log information as usual, but will additionally include a URL that you can navigate to from a client on the same network to see how the site renders.
+
+```
+You can now view gatsbyjs.org in the browser.
+⠀
+  Local:            http://0.0.0.0:8000/
+  On Your Network:  http://192.168.0.212:8000/ // highlight-line
+```
+
```

Comment:
After my testing it seems like this could be shipped after adding note that you can't visit `0.0.0.0:8000` on Windows (but will work with either `localhost:8000` or "On Your Network" URL.

And then separately work on 0.0.0.0 + Windows displaying localhost instead and making 0.0.0.0 default?

Reply:
Sounds good to me!